app/services/random_quotes.js

```diff
@@ -0,0 +1,11 @@
+const { getRequest } = require('../helpers/http_requests');
+const { defaultError } = require('../errors');
+
+exports.getRandomQuote = async () => {
+  try {
+    const result = await getRequest('https://geek-jokes.sameerkumar.website/api?format=json');
```

Comment:
The url for the API should be an env variable

Reply:
And the env variable should be configured in the `config/index.js` file

Reply:
The suggestion implemented.